extensions/adapters/raster/src/main/java/mil/nga/giat/geowave/adapter/raster/RasterUtils.java

```diff
@@ -564,7 +564,7 @@ public static GridCoverage2D mosaicGridCoverages(
 		}
 		final double scaleX = rescaleX * (width / imageWidth);
 		final double scaleY = rescaleY * (height / imageHeight);
-		if ((scaleX != 1) || (scaleY != 1)) {
+		if ((Math.abs(scaleX - 1) > SIMPLIFICATION_MAX_DEGREES) || (Math.abs(scaleY - 1) > SIMPLIFICATION_MAX_DEGREES)) {
```

Comment:
SIMPLIFICATION_MAX_DEGREE is being used for another purpose, we probably just want a constant for DOUBLE_PRECISION_EPSILON_EQUALS or something like that defined in a utility class common to everything and re-used everywhere so we don't have various constants throughout the code